automation_tools/__init__.py

```diff
@@ -3173,6 +3173,67 @@ def attach_subscription(url=None, consumer=None, pool_id=None, count=None):
         raise ValueError("Pass values under pool_id and count")
 
 
+def refresh_manifest(url=None, consumer=None):
+    """Task to remove all subscriptions from manifest and attach required ones.
+
+    The following environment variables affect this command:
+
+    SM_URL
+      Subscription Manager URL (e.g. 'https://subscription.rhsm.redhat.com')
+    CONSUMER
+        A consumer hash to be used for getting the manifest
+    RHN_USERNAME
+        Red Hat Network username
+    RHN_PASSWORD
+        Red Hat Network password
+
+    :param url: Subscription Manager URL
+    :param consumer: A consumer hash to be used for getting the manifest
+    :returns: boolean True/False
+    """
+    user = os.environ.get('RHN_USERNAME')
```

Comment:
please avoid using environment variables. We are moving away from using them https://github.com/SatelliteQE/automation-tools/issues/799

Reply:
Okay I was not aware about that..
As this requires to remove from other functions as well hence would be good to handle separately or under #799

Reply:
I think it would be a easy fix here itself, just add these to parameter, and in robottelo-ci call it with those parameter. Those are already in the job. 
Not much work involved in the job as job is already a pipeline job if I am not wrong. 

Also, Is this a urgent PR?

Reply:
thanks .. removed the env vars deps now in latest commit
Also yeah it's not a critical PR but would be great to merge asap as avoid further noises around manifest download.